### ISSUE TITLE
cross-compilation/installation via buildroot

### ISSUE THREAD

**Issue by PhoGGy** [panda-official/TimeSwipe.py#8]
timeswipeOS is based on buildroot.
I need to be able to either cross compile or install the binary wheel from inside buildroot for the target.
Crosscompilation did not work as the node-gyp paths would have to be changed from /usr/lib to $(TARGET_DIR)/usr/lib if its compiled by buildroot. Have a look at my fork for an example.
Now installation of the binary wheel is not as easy as I expected as well. I can not not cross-run pip3 for the target system on the host. Therefore as a workaround a copied the wheel into the target directory and created a first-boot-script to run pip3 install on first boot. Unfortunately that didn't work either, as it then wanted to download cmake_setuptools package.
Why does the binary depend on CMake_setuptools? I need to find an installation method, that does not require to download any further packages after deployment.
Thanks and Greetings
Phil
PANDA GmbH

**Comment by iluxa** [comment]
You are right, cmake_setuptools dependency is not required even for build from sources.
New release has no this dependency: [v0.0.10](https://github.com/panda-official/TimeSwipe.py/releases/tag/v0.0.10)

**Comment by IngoKaiser** [comment]
@PhoGGy please review and close when working
